packages/internal/src/config.js

```diff
@@ -13,6 +13,26 @@
  *   package are placed here.
  */
 
+/**
+ * ClusterName specifies a collection of networks.
+ * The specific networks associated with a particular name may vary from context
+ * to context, and may also overlap (e.g. a "local" cluster may connect to the
+ * same remote networks as a "testnet" cluster), but this type nevertheless
+ * supports cross-package coordination where the values associated with static
+ * labels are subject to choice of cluster. Some examples:
+ *   - the chain ID for a static label like "Agoric" or "Ethereum"
+ *   - the cryptographic hash for a static label like "BLD" or "USDC"
+ *   - the URL for a service like "Agoric RPC" or "Axelar" or "Spectrum"
+ *
+ * "mainnet" should always include the Agoric network described by
+ * https://main.agoric.net/network-config rather than some other alternative.
+ *
+ * XXX This actually belongs somewhere else, possibly with refactoring.
```

Comment:
given that it belongs somewhere else, putting it in `@agoric/internal` is worse to me than client-utils. It's now available to everything and they'll key off these loosely defined values. I don't understand the compulsion to move it any lower than portfolio-api but I don't want to delay the PR on aligning on this.